### ISSUE TITLE
Feature request: "Photo Overlays" date picker (today box)

### ISSUE THREAD

**Issue by sun-geo** [openstreetmap/iD#8605]
A usual task while editing the map is to look for the latest street picture (especially if there are many pictures tracks available for a certain map area).
So I'm starting to enter the date from today and afterwards i go back step by step the month or the year.

It would be nice either
* the date from today is preset for the "From"-line or 
* adding a checkbox with a function which automaticallyenters the date from today for the "From"-line

![before-after](https://user-images.githubusercontent.com/47290585/127498086-f16aa693-083d-4a1c-a38f-2af794761e4b.png)

**Comment by nickplesha** [comment]
@sun-geo The datepicker on the date input field already has a today button (depending on the browser):
<img width="422" alt="Screenshot 2021-08-05 at 19 04 52" src="https://user-images.githubusercontent.com/3823786/128391716-ac71e704-a3b0-4948-940f-543df0a894a3.png">
What browser are you using and what happens when you click the date field?

**Comment by boothym** [comment]
>A usual task while editing the map is to look for the latest street picture

I agree, when there's lots of imagery I want to see this as well. However I'm not sure a "today" box would be helpful - as surely it would be highly unlikely to show any images? You would still need to click the date picker again and go back a month.

What I normally do just now is type in 1/1/2021 so that limits it to imagery from this year / 6 months ago.

Perhaps a dropdown with 1 month / 6 months / 12 months / 18 months / 2 years etc would be better?

**Comment by tyrasd** [comment]
since the date range selection for street level photos is now done via a date slider, so the this idea should be obsolete.

<img width="333" src="https://github.com/user-attachments/assets/321fece7-1058-482c-b6de-815360e90c00" />
